config/advanced.yaml

```diff
@@ -12,6 +12,7 @@ data:
 mapping:
   voxel_size: 1.0 # <- optional
   max_points_per_voxel: 20
+  max_threads: -1
```

Comment:
Not sure if this is still part of the draft, but as a user I would wonder if the number of threads is only limited for mapping.

How about using `max_threads` as "first-level" key like `out_dir`?

Reply:
Yeah, this is a draft, but it's clearly confusing. Reading this makes me feel I will have multiple maps running in parallel :robot: